libs/designer-ui/src/lib/floatingactionmenu/index.tsx

```diff
@@ -132,9 +156,42 @@ export const FloatingActionMenu = (props: FloatingActionMenuProps): JSX.Element
     }
   };
 
-  const handleMenuItemSelected = (_item: FloatingActionMenuItem) => {
+  const enterVariableNamePlaceholder = intl.formatMessage({
+    defaultMessage: 'Enter variable name',
+    description: 'Placeholder for variable name for new dynamically added parameter',
+  });
+
+  const addNewDynamicallyAddedParameter = (item: FloatingActionMenuItem) => {
+    const { icon, type, placeholder } = item;
+    const format =
+      type === DynamicallyAddedParameterType.Date || type === DynamicallyAddedParameterType.Email ? type.toLowerCase() : undefined;
+    dynamicParameterProps.push({
+      icon,
+      schemaKey: guid(), // TODO: generate schemaKey
+      properties: {
+        description: placeholder,
+        format,
+        title: enterVariableNamePlaceholder, // TODO: generate title based on schemaKey
```

Comment:
The 1) property key here (called `schemaKey` in this code) and 2) `title` are named/generated differently from v1 designer (BPMUX repo).

In BPMUX, instead of a guid the keys would be named -- text, text_1, text_2, etc.
and the `title` would be generated based on those, for example -- "Text", "Text 1", "Text 2", etc.

Reply:
Questions:
1) Is this functionality we can do without for now?
2) What repercussions does this have if we want to load a flow saved in v3 designer in v1 designer?

Reply:
Right now, this is not blocking -- we're in a time crunch so would like to see this PR merged.